### ISSUE TITLE
2.1 PPP 90 Minutes

### ISSUE THREAD

**Issue by anish-kum** [anish-kum/PPP-Spark-2025-2026#25]
90 min PPP

**Comment by anish-kum** [comment]
<img width="1279" height="548" alt="Image" src="https://github.com/user-attachments/assets/9384d1a0-721a-4aba-a7e3-8ff9490bdd62" />
----------------------------------------------------------------------------------------------------------------------------------------------

What I learned:

- Convolutional Neural Networks (CNNs): Overview of CNNs as powerful models for image processing tasks, leveraging convolutional layers.

- Pooling Layers: Discusses the role of pooling layers in reducing spatial dimensions and retaining essential features.

- Batch Normalization: Introduces batch normalization to stabilize and accelerate training by normalizing layer inputs.

- Dropout Regularization: Explains dropout as a technique to prevent overfitting by randomly dropping units during training.

- Transfer Learning: Highlights the use of pre-trained models to improve performance on new tasks with limited data.

- Model Evaluation: Emphasizes the importance of evaluating CNNs using metrics like accuracy and loss on validation datasets.

----------------------------------------------------------------------------------------------------------------------------------------------

2 Questions I still had:

-What problem does pooling solve in a CNN, and what might happen if you remove pooling layers completely?

-Why does dropout help stop a model from overfitting during training?

----------------------------------------------------------------------------------------------------------------------------------------------

Point of Clarification: You asked last time whether I used AI for these summaries. Within Udacity, which is where I take this course, they have a nice AI feature to summarize what you have reviewed (which in this case is 5.26-5.31), so I use that alongside other course context. For the questions, I utilize what I remember + the summary.

**Comment by drewmca-pkwy** [comment]
@anish-kum Thank you for the clarification, Anish! I'm glad you're moving through this content. I'd like one change for this next turn-in: For the summary, I'd like you to take the six elements that AI generates and create _three_ that you write out yourself. 

For each of those three bullet points, I'd like you to:
- tell me what the concept or topic is
- tell me why that topic or concept is important to you

The important part of these cards is not so I know exactly what content you have covered. I need to know what parts of that content has "sunk in" and made a difference on your learning.
